example/ios/example.xcodeproj/project.pbxproj

```diff
@@ -60,6 +61,8 @@
 		4C3978FD8F0B224CAF71A707 /* Pods-example-tvOSTests.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-example-tvOSTests.release.xcconfig"; path = "Target Support Files/Pods-example-tvOSTests/Pods-example-tvOSTests.release.xcconfig"; sourceTree = "<group>"; };
 		7B170452E329916E4D48E91C /* Pods-example.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-example.release.xcconfig"; path = "Target Support Files/Pods-example/Pods-example.release.xcconfig"; sourceTree = "<group>"; };
 		81AB9BB72411601600AC10FF /* LaunchScreen.storyboard */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = file.storyboard; name = LaunchScreen.storyboard; path = example/LaunchScreen.storyboard; sourceTree = "<group>"; };
+		94839A952606E7E8006AE771 /* example-Bridging-Header.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = "example-Bridging-Header.h"; sourceTree = "<group>"; };
```

Comment:
Can we call the bridging header somethine like `RNHealth-Bridging-Header.h` instead of `example-`?

Reply:
hey, thank you for your comment. 

This bridging header was included as a workaround for an issue(https://github.com/agencyenterprise/react-native-health/pull/67#issuecomment-803712374) that I'm currently trying to fix. This header is related to the example project of this repository, not the actual Pod. This bridging header (and Void.swift file) probably will be removed before this PR is complete.

Note that I included `RCTAppleHealthKit-Bridging-Header.h` which is the bridging header that actually "bridges" our objc-c code with swift.

Reply:
I tried to give a better explanation of this issue I mentioned in my latest comment, could you take a look? 😄